### PR TITLE
Q1 2019 patch release

### PR DESCRIPTION
**Pull request by jeff-matthews**
## This PR is a:

Content update

## Summary

When this pull request is merged, it will merge all Q1 2019 release-related changes.

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=4032) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=4032) before we can accept your contribution.<br/>**4** out of **5** committers have signed the CLA.<br/><br/>:white_check_mark: jeff-matthews<br/>:white_check_mark: keharper<br/>:white_check_mark: nuzil<br/>:white_check_mark: dobooth<br/>:x: Jeanne Frontain<hr/>**Jeanne Frontain** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

File: _includes/config/es-elasticsearch-magento.md [hunk 1]
```diff
@@ -1,102 +1,97 @@
...
+|**Elasticsearch Index Prefix**|Enter the Elasticsearch index prefix. If you use a single Elasticsearch instance for more than one Magento installation (Staging and Production environments), you must specify a unique prefix for each installation. Otherwise, you can use the default prefix magento2.|
+|**Enable Elasticsearch HTTP Auth**|Click **Yes** only if you enabled authentication for your Elasticsearch server. If so, provide a username and password in the provided fields.|
+{:style="table-layout:auto;"}
+
+{:start="4"}
+1. Click **Test Connection**.
+
+You will see either:
+
+<img src="{{ site.baseurl }}/common/images/elastic_test-success.png">
```
Comment (dshevtsov) [inline _includes/config/es-elasticsearch-magento.md]: should be in kramdown

File: _includes/config/es-elasticsearch-magento.md [hunk 2]
```diff
@@ -1,102 +1,97 @@
...
+
+Continue with:
+
+* [Configure Apache and Elasticsearch]({{ page.baseurl }}/config-guide/elasticsearch/es-config-apache.html)
+* [Configure nginx and Elasticsearch]({{ page.baseurl }}/config-guide/elasticsearch/es-config-nginx.html)
+
+or you will see:
+
+<img src="{{ site.baseurl }}/common/images/elastic_test-fail.png">
```
Comment (dshevtsov) [inline _includes/config/es-elasticsearch-magento.md]: should be in kramdown

File: _includes/config/es-elasticsearch-magento.md [hunk 3]
```diff
@@ -1,102 +1,97 @@
...
+
+* Make sure the Elasticsearch server is running.
+* If the Elasticsearch server is on a different host from Magento, log in to the Magento server and ping the Elasticsearch host. Resolve network connectivity issues and test the connection again.
+* Examine the command window in which you started Elasticsearch for stack traces and exceptions. You must resolve those before you continue.
+In particular, make sure you started Elasticsearch as a user with `root` privileges.
+* Make sure that [UNIX firewall and SELinux]({{ page.baseurl }}/config-guide/elasticsearch/es-overview.html#firewall-selinux) are both disabled, or set up rules to enable Elasticsearch and Magento to communicate with each other.
+* Verify the value of the **Elasticsearch Server Hostname** field. Make sure the server is available. You can try the server's IP address instead.
+* Use the command `netstat -an | grep **listen-port**` command to verify that the port specified in the **Elasticsearch Server Port** field is not being used by another process.
+  For example, to see if Elasticsearch is running on its default port, use the following command:
+  `netstat -an | grep 9200`
```
Comment (dshevtsov) [inline _includes/config/es-elasticsearch-magento.md]: should be wrapped in ` ```bash `

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**